backend/src/main/java/asegroup1/api/models/LandRegistryQuerySelect.java

```diff
@@ -0,0 +1,231 @@
+package asegroup1.api.models;
+
+/**
+ * 
+ * @author Rikkey Paal
+ */
+public class LandRegistryQuerySelect {
+
+	private boolean propertyType;
+	private boolean estateType;
+	private boolean transactionDate;
+	private boolean pricePaid;
+	private boolean newBuild;
+	private boolean transactionCategory;
+	private boolean paon;
+	private boolean saon;
+	private boolean street;
+	private boolean locality;
+	private boolean town;
+	private boolean district;
+	private boolean county;
+	private boolean postcode;
+	private boolean date;
+	private boolean housePriceIndex;
+
+	public LandRegistryQuerySelect(boolean propertyType, boolean estateType, boolean transactionDate, boolean pricePaid, boolean newBuild, boolean transactionCategory,
+			boolean paon, boolean saon, boolean street, boolean locality, boolean town, boolean district, boolean county, boolean postcode, boolean date, boolean housePriceIndex) {
+		this.propertyType = propertyType;
+		this.estateType = estateType;
+		this.transactionDate = transactionDate;
+		this.pricePaid = pricePaid;
+		this.newBuild = newBuild;
+		this.transactionCategory = transactionCategory;
+		this.paon = paon;
+		this.saon = saon;
+		this.street = street;
+		this.locality = locality;
+		this.town = town;
+		this.district = district;
+		this.county = county;
+		this.postcode = postcode;
+		this.date = date;
+		this.housePriceIndex = housePriceIndex;
+	}
+
+	public LandRegistryQuerySelect() {
+		this(false, false, false, false, false, false, false, false, false, false, false, false, false, false, false, false);
+	}
+
+	public String buildQuerySelect() {
+		String str = "SELECT ";
+		if (isPropertyType()) {
+			str += "?propertyType ";
```

Comment:
This should never been done if multiple things will be appended to String, as string is an immutable object. A `StringBuilder` should be used instead.

Reply:
Done